core/src/main/scala/cats/syntax/unorderedFoldable.scala

```diff
@@ -0,0 +1,33 @@
+package cats
+package syntax
+
+import cats.instances.long._
+
+trait UnorderedFoldableSyntax extends UnorderedFoldable.ToUnorderedFoldableOps {
+  implicit final def catsSyntaxUnorderedFoldableOps[F[_]: UnorderedFoldable, A](fa: F[A]): UnorderedFoldableOps[F, A] =
+    new UnorderedFoldableOps[F, A](fa)
+}
+
+final class UnorderedFoldableOps[F[_], A](val fa: F[A]) extends AnyVal {
+  /**
+    * Count the number of elements in the structure that satisfy the given predicate.
+    *
+    * For example:
+    * {{{
+    * scala> import cats.implicits._
+    * scala> val set1 = Set[String]()
+    * scala> set1.count_(_.length > 0)
+    * res0: Long = 0
+    *
+    * scala> val set2 = Set("hello", "world", "!")
+    * scala> set2.count_(_.length > 1)
+    * res1: Long = 2
+    *
+    * scala> val set3 = Set(41, 32, 23)
+    * scala> set3.count_(_ % 2 == 0)
+    * res2: Long = 1
+    * }}}
+    */
+  def count_(p: A => Boolean)(implicit F: UnorderedFoldable[F]): Long =
```

Comment:
Maybe it'd be better to have `count` defined in the `UnorderedFoldable` trait itself, and only call `count` here? Otherwise you can only use it with the syntax implicits, and not just `F.count`

Reply:
You can't do that if you want to maintain binary compat.

Reply:
Aww damn :( the encoding of traits strikes again

Reply:
we should be able to define extensions for typeclass instances, like:

```scala
implicit class UFOps[F[_]](val instance: UnorderedFoldable[F]) extends AnyVal {
  def count(...)
}
```